### ISSUE TITLE
[Feature Request] Add invert tickbox in Ports table 

### ISSUE THREAD

**Issue by Dropsleutel** [betaflight/betaflight-configurator#1900]
It would be convenient to invert the Rx and Tx lines from each UART from the port configuration table, instead of CLI.

**Comment by mikeller** [comment]
@Dropsleutel: Agreed. The problem with this is that currently, inversion is configured per protocol, and not per port. The idea is to fix this in a rewrite of how ports are configured, and then add the configuration to the UI.

**Comment by Dropsleutel** [comment]
Thanks it's not urgent, more a luxury :)

**Comment by haslinghuis** [comment]
Now CRSF/ERLS are dominant - think this is no longer a priority.
